packages/wrangler/src/api/upgrade.ts

```diff
@@ -0,0 +1,94 @@
+import { writeFileSync, existsSync } from "fs";
```

Comment:
Can this file use async versions of `fs` and `execa`?

Reply:
I prefer not to.

Reply:
Why not, out of interest?

Reply:
It doesn't need to be `async` there are no other operations or commands occurring that could be running why these things happen and I would just end up using `await` blocking the thread anyway.